dali/kernels/slice/slice_flip_normalize_permute_gpu.cuh

```diff
@@ -26,144 +26,27 @@
 #include "dali/kernels/common/copy.h"
 #include "dali/kernels/kernel.h"
 #include "dali/kernels/slice/slice_flip_normalize_permute_common.h"
-
+#include "dali/kernels/slice/slice_flip_normalize_permute_pad_kernel.cuh"
 namespace dali {
 namespace kernels {
 
```

Comment:
now that you've removed the CUDA code from here, this file can be a *.cc file, I believe